src/Symfony/Component/Form/Test/DeprecationErrorHandler.php

```diff
@@ -24,7 +24,11 @@ public static function handle($errorNumber, $message, $file, $line, $context)
             return true;
         }
 
-        return \PHPUnit_Util_ErrorHandler::handleError($errorNumber, $message, $file, $line);
+        if (class_exists('\PHPUnit_Util_ErrorHandler')) {
+            return \PHPUnit_Util_ErrorHandler::handleError($errorNumber, $message, $file, $line);
+        }
+
+        return \PHPUnit\Util\ErrorHandler::handleError($errorNumber, $message, $file, $line);
```

Comment:
You mean just for the namespaced versions? I was unsure about it and therefore kept it as is. I can easily update that if wanted.